test/integration/simple_example/inspec.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: my-profile
+title: GCP Cloud DNS InSpec Profile
+maintainer: Eric Fouarge
+copyright: Google LLC
+copyright_email: ericalan@google.com
+license: Apache-2.0
+summary: An InSpec Compliance Profile For Spinnaker in GCP
+version: 0.1.0
+inspec_version: '>= 2.3.5'
+attributes:
+- name: gcp_project_id
```

Comment:
Subjective: I personally prefer using consistent names for things like `project_id` everywhere, so we don't have to map from `project_id` in Terraform `gcp_project_id` in our tests. This approach also helps out if we use kitchen-terraform since we'll have a 1:1 mapping between Terraform variables/outputs and InSpec attributes. This point's pretty bikeshed-y though, so do what you think is appropriate.